metrics/cli/docs/04_managing_firestore_data.md

```diff
@@ -0,0 +1,96 @@
+# Managing Firestore data
+
+## Motivation
+> What problem is this project solving?
+
+Since, the Metrics App stores [allowed domains](https://github.com/platform-platform/monorepo/blob/master/docs/19_security_audit_document.md#the-allowed_email_domains-collection) and [feature config](https://github.com/platform-platform/monorepo/blob/master/docs/19_security_audit_document.md#the-feature_config-collection) collections in Cloud Firestore and these collections are protected by the Firebase rules from the reading and writing, so we should find an approach of managing Firestore data using [Admin SDK](https://developers.google.com/admin-sdk).
```

Comment:
```suggestion
Since the Metrics App stores [allowed domains](https://github.com/platform-platform/monorepo/blob/master/docs/19_security_audit_document.md#the-allowed_email_domains-collection) and [feature config](https://github.com/platform-platform/monorepo/blob/master/docs/19_security_audit_document.md#the-feature_config-collection) collections in Cloud Firestore and these collections are protected by the Firebase rules from the writing, we should find an approach of managing Firestore data using [Admin SDK](https://developers.google.com/admin-sdk).
```